src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -143,12 +143,24 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
             throw new EntityDoesNotExistException(
                     "Trying to get feedback sessions for a course that does not exist.");
         }
+        return getFeedbackSessionsForUserInCourseSkipCheck(courseId, userEmail);
+    }
 
+    /**
+     * @param courseId
+     * @param userEmail
+     * @return a list of viewable feedback sessions for any user for his course.
+     * @throws EntityDoesNotExistException
+     */
```

Comment:
The comment doesn't mention anything about the 'skipTest' part. In any case, I try to avoid this kind of method variations. They are a sign that the code is getting messy.